components/automate-cli/pkg/verifyserver/server/api/v1/opensearchbackups3.go

```diff
@@ -0,0 +1,28 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) CheckOSBackupS3(c *fiber.Ctx) {
+
+	req := new(models.S3BackupDetails)
+
+	if err := c.BodyParser(req); err != nil {
+		errString := fmt.Sprintf("Invalid request for S3 backup check: %v", err.Error())
+		h.Logger.Error(fmt.Errorf(errString))
+		c.Status(fiber.StatusBadRequest).JSON(errString)
```

Comment:
c.Next(&fiber.Error{Code: http.StatusBadRequest, Message: err.Error()})
We are returning error this way for all checks